R/RunDiagnostics.R

```diff
@@ -747,7 +745,7 @@ runCohortDiagnostics <- function(packageName = NULL,
   )
 
   # Add all to zip file -------------------------------------------------------------------------------
-  writeResultsZip(exportFolder, databaseId, vocabularyVersion, vocabularyVersionCdm)
+  writeResultsZip(exportFolder, databaseId, vocabularyVersion, vocabularyVersion)
```

Comment:
why was vocabularyVersionCdm changed to vocabularyVersion ?

Reply:
Both these last two parameters are actually no longer used since the recent meta-data patch. I will remove them.